cli/command/trust/info.go

```diff
@@ -101,9 +101,48 @@ func lookupTrustInfo(cli command.Cli, remote string) error {
 	}
 	printSignatures(cli, remote, signatureRows)
 
+	roleWithSigs, err := notaryRepo.ListRoles()
+	if err != nil {
+		return trust.NotaryError(ref.Name(), err)
+	}
+
+	signerRoleToKeyIDs, adminRoleToKeyIDs := getSignerAndBaseRolesWithKeyIDs(roleWithSigs)
+
+	fmt.Fprintf(cli.Out(), "\nList of signers and their KeyIDs:\n\n")
+	printRolesWithKeyIDs(cli, signerRoleToKeyIDs)
+
+	fmt.Fprintf(cli.Out(), "\nList of admins and their KeyIDs:\n\n")
+	printRolesWithKeyIDs(cli, adminRoleToKeyIDs)
+
 	return nil
 }
 
+// Extract signer keys and admin keys from the list of roles
+func getSignerAndBaseRolesWithKeyIDs(roleWithSigs []client.RoleWithSignatures) (map[string][]string, map[string][]string) {
```

Comment:
hm this comment never posted for some reason on my last review, but maybe this should be renamed to `getSignerAndAdminRolesWithKeyIDs`?  This is the last nit I was referring to (the test should get updated too)